modules/openfast-library/src/FAST_Subs.f90

```diff
@@ -3394,6 +3521,322 @@ end subroutine cleanup
+   CALL WrScr('')
+
+!>>>>>>>>>>>>>>>>>>>> 
+   ! -------------------------------------------------------------
+   ! READ FROM THE PRIMARY OPENFAST (TIME-DOMAIN) INPUT FILE
+   ! do this before reading the rest of the variables in this
+   ! steady-state input file so that we don't accidentally 
+   ! overwrite them.
+   ! -------------------------------------------------------------
+   IF ( PathIsRelative( FstFile ) ) FstFile = TRIM(PriPath)//TRIM(FstFile)
+   CALL FAST_ReadPrimaryFile( FstFile, p, m_FAST, .true., ErrStat2, ErrMsg2 )
+      CALL SetErrStat( ErrStat2, ErrMsg2, ErrStat, ErrMsg, RoutineName)
+      if ( ErrStat >= AbortErrLev ) then
+         call cleanup()
+         RETURN
+      end if
+   
+   !--------------------------------------------
+   ! Overwrite values for parameters that we do not
+   ! want to read from the input file:
+   !--------------------------------------------
+   p%DT   = 1.0_DbKi ! we'll make this unity to represent case numbers instead of time in the case of AeroMap generation
+   p%TMax = p%DT ! overwrite this later when we have the number of cases to run
+   p%InterpOrder = 1 ! set this to 1 so we have two copies of inputs in the solver
+   p%NumCrctn = 0
+   p%DT_UJac = 9999.0_ReKi ! any non-zero number will do ! maybe we will want to use this????
+   p%n_SttsTime = 1
+   p%n_ChkptTime = HUGE(p%n_ChkptTime)
+
+   p%CompInflow = Module_NONE
+   p%CompServo = Module_NONE
+   p%CompHydro = Module_NONE
+   p%CompSeaSt = Module_NONE
+   p%CompSub = Module_NONE
+   p%CompMooring = Module_NONE
+   p%CompIce = Module_NONE
+   if ( p%CompAero /= Module_AD) then
+      p%CompAero = Module_AD
+      call WrScr('Warning: AeroDyn must be used for generating AeroMaps. Check that variable "AeroFile" is set properly in the OpenFAST input file.')
+   end if
+   if (p%CompElast == Module_BD) then
+      CALL SetErrStat( ErrID_Warn, "AeroMaps with BeamDyn have not been verified.", ErrStat, ErrMsg, RoutineName)
+   end if
+   
+   p%DT_Out = p%DT
+   p%n_DT_Out = 1 ! output every step (i.e., every case)
+   p%TStart = 0.0_DbKi
+
+   p%Linearize = .false. ! we use p%CompAeroMaps to do a subset of the linearization routines
+   p%CalcSteady = .false.
+   p%TrimCase = TrimCase_none
+   p%NLinTimes = 1
+   p%LinInputs = LIN_ALL
+   p%LinOutputs = LIN_ALL
+   
+   p%LinOutMod = .TRUE.    ! if debugging, this will allow us to output linearization files (see parameter "output_debugging" in FAST_SS_Solver.f90); otherwise this doesn't do anything
+   p%LinOutJac = .TRUE.    ! if debugging, this will allow us to output linearization files (see parameter "output_debugging" in FAST_SS_Solver.f90); otherwise this doesn't do anything
+   p%WrVTK = VTK_None
+   p%VTK_Type = VTK_None
+   p%n_VTKTime = 1
+   m_FAST%Lin%FoundSteady = .false.
+   p%LinInterpOrder = p%InterpOrder ! 1 ! always use linear (or constant) interpolation on rotor 
+   !--------------------------------------------
+!<<<<<<<<<<<<<<<<<<<<<<<<
```

Comment:
maybe remove/ adapt this comment line to avoid mixing it up with a merge conflict

Reply:
Done.